master_config.json

```diff
@@ -121,6 +121,12 @@
         "symbol": "EURC",
         "mint": "HzwqbKZw8HxMN6bF2yFZNrht3c2iXXzpKcFu7uBEDKtr",
         "logoUri": "https://www.circle.com/hubfs/Brand/EURC/EURC-icon_128x128.png"
+      },
+      {
+        "name": "dogwifhat",
+        "symbol": "WIF",
+        "mint": "EKpQGSJtjMFqKZ9KQanSqYXRcF8fBopzLHYxdM65zcjm",
+        "logoUri": "https://bafkreibk3covs5ltyqxa272uodhculbr6kea6betidfwy3ajsav2vjzyum.ipfs.nftstorage.link/"
```

Comment:
is there a more robust URL?

Reply:
that one is solscans
could also use coinmktcap
https://s2.coinmarketcap.com/static/img/coins/64x64/28752.png

Reply:
yeah this one is fine, cmc so small